src/quo2/components/account_selector.cljs

```diff
@@ -0,0 +1,80 @@
+(ns quo2.components.account-selector
+  (:require
+   [quo.theme :as theme]
+   [quo.react-native :as rn]
+   [quo2.foundations.colors :as colors]))
+
+(def themes
+  {:light {:default         {:bg            colors/neutral-10
+                             :account-text  colors/black
+                             :label-text    colors/neutral-50}
+           :transparent     {:bg            colors/neutral-80-opa-5
+                             :account-text  colors/black
+                             :label-text    colors/neutral-50}}
+
+   :dark  {:default         {:bg            colors/neutral-80-opa-80
+                             :account-text  colors/white
+                             :label-text    colors/neutral-40}
+           :transparent     {:bg            colors/white-opa-5
+                             :account-text  colors/white
+                             :label-text    colors/neutral-40}}})
+
+;we prefer using temp uri for assets so that un-neccessary assets do not become a part of release bundle
+(def temp-fire-icon-uri "https://s3-alpha-sig.figma.com/img/58f0/9735/781a2f7ece5e5c0fde2cb7e9d2c22c77?Expires=1661731200&Signature=f5hvtb9Wz1Zwd4-dft9gOyBP25rR1wrVFtHyCQh9NCslCBZ7JJzex7qYdYZn2Qvf6ep50HIdkO1oh6wVBqeds1BrX0T4ojSzM8HA0CQkXlVveEwEyjcqb~XPx3m2zMFEcaoUX87Ho3I6ZHK477WwoHgKX9JwfpfoNZBSqVT1TiTiTPpNdv4AEY-5TJcuZk7mS1-O1eCuEoQA-VTWX3UXExsZ9wip-rnrp2o8a7dx1run9jHSNqsKfEMgK2Mc3zjphzW34asgrxCf1plRv3ZD12ht~pzsuph8FeeBb9HFLtumbUKbDI7L6sffEFGY6msxUBr0ret0akFlk1x00gnwCA__&Key-Pair-Id=APKAINTVSUGEWH5XD5UA")
+
+(defn account-container-row [background-color]
+  {:height 40
+   :flex-direction :row
+   :align-items :center
+   :background-color background-color
+   :border-radius 12
+   :width 355})
```

Comment:
using `:width` might give different results across different devices (in this case, as container is covering screen width), better to use margins, padding's. wdyt?

Reply:
yes much better that way, I shall refactor

Reply:
margin padding was not providing desired result since we have external padding coming from parent component, 
I think component width should be independent of such scenarios, I am going with the following formula -> (device width - 80) because design shows margin of 40 on each side (left and right)

Reply:
> margin padding was not providing desired result since we have external padding coming from parent component.

Yes, that's what I meant. Keep `:width` dynamic, as same as `:height` and use those padding's on parent.

Reply:
@Parveshdhull : Thanks for the advice